packages/core/core/src/PackagerRunner.js

```diff
@@ -66,16 +68,43 @@ export default class PackagerRunner {
         };
   }
 
-  async writeBundle(bundle: InternalBundle, bundleGraph: InternalBundleGraph) {
+  async writeBundles(bundleGraph: InternalBundleGraph) {
+    let farm = nullthrows(this.farm);
+    // TODO: maybe avoid doing this if we don't have to?
+    let {ref, dispose} = await farm.createSharedReference(bundleGraph);
```

Comment:
For cases where there are less bundles than the amount of workers, we are sending the bundle graph across IPC slightly more than we need to. Not sure if this is that much of a concern since in those cases the BundleGraph would reasonably small?

Reply:
could be one really big bundle or something... seems ok though since it will only get serialized once